src/ImageSharp/MetaData/ImageMetaData.cs

```diff
@@ -116,14 +118,12 @@ public double VerticalResolution
         /// </summary>
         public ExifProfile ExifProfile { get; set; }
 
-        /// <summary>
-        /// Gets or sets the frame delay for animated images.
-        /// If not 0, this field specifies the number of hundredths (1/100) of a second to
-        /// wait before continuing with the processing of the Data Stream.
-        /// The clock starts ticking immediately after the graphic is rendered.
-        /// </summary>
+        /// <inheritdoc/>
         public int FrameDelay { get; set; }
 
+        /// <inheritdoc/>
```

Comment:
Ugh, just realised that VS doesn't currently support intellisense based on inheritdoc. I don't want to stop using it though as it massively cuts down on errors duplicating XML docs and Wyam already supports it for our API docs. 

It looks like MVC are planning to use post processing to augment their XML output. I think we should follow suite. 
https://github.com/aspnet/Mvc/issues/928